### PR TITLE
Add legacy links to all meta page routes

### PR DESCRIPTION
**Pull request by tomas862**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | for some routes legacy links were missing which possibly could cause an issues when generating legacy links
| Type?         | bug fix
| Category?     | BO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | 
| How to test?  | No Q/A testing are required here

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/12787)

### CODE REVIEW

File: src/PrestaShopBundle/Resources/config/routing/admin/configure/shop_parameters/meta.yml [hunk 1]
```diff
@@ -12,45 +12,52 @@ admin_metas_search:
...
     _legacy_controller: AdminMeta
+    _legacy_link: AdminMeta:updatemeta
 
 admin_metas_delete:
   path: /delete/{metaId}
   methods: DELETE
   defaults:
     _controller: 'PrestaShopBundle:Admin\Configure\ShopParameters\Meta:delete'
     _legacy_controller: AdminMeta
+    _legacy_link: AdminLanguages:deletemeta
```
Comment (matks) [inline src/PrestaShopBundle/Resources/config/routing/admin/configure/shop_parameters/meta.yml]: `AdminLanguages` or `AdminMeta` ?

Comment (tomas862) [reply src/PrestaShopBundle/Resources/config/routing/admin/configure/shop_parameters/meta.yml]: AdminMeta ofcourse :open_mouth:

**Comment by tomas862** [comment]
@matks done :rofl:

**Comment by matks** [comment]
@tomas862 Since QA cannot test this PR, can you delegate validation to Travis by adding 2 or 3 tests in LegacyUrlConverterTest for these routes ? 😉

**Comment by tomas862** [comment]
Thanks @matks - feeling much better now with tests :smile:

**Review by matks — APPROVED** [review]

**Comment by matks** [comment]
Thanks @tomas862 !

**Pull request merged**